parsons/databases/bigquery/__init__.py

```diff
@@ -0,0 +1,3 @@
+from parsons.databases.bigquery.bigquery import BigQuery
```

Comment:
Went back and forth on which organizational convention to follow. Decided to move to the databases folder but no strong opinion here if ppl want to leave in the google folder

Reply:
I prefer this way. I think it makes more sense to be in databases.